src/main/msp/msp.c

```diff
@@ -3359,11 +3359,13 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
                 disableRunawayTakeoff = sbufReadU8(src);
             }
             if (command) {
+#ifndef SIMULATOR_MULTITHREAD //In simulator mode we can safely arm with MSP link.
                 mspArmingDisableByDescriptor(srcDesc);
                 setArmingDisabled(ARMING_DISABLED_MSP);
                 if (ARMING_FLAG(ARMED)) {
                     disarm(DISARM_REASON_ARMING_DISABLED);
                 }
+#endif
```

Comment:
We should allow arming with MSP in SITL.

Reply:
@ledvinap I am not sure what is the other side, the code can arm with MSP now.

Reply:
Sorry, I don't know exactly how MSP works. But there is command to disable arming (MSP_SET_ARMING_DISABLED with nonzero argument) and enable arming (MSP_SET_ARMING_DISABLED zero argument).
So I would expect that `MSP_SET_ARMING_DISABLED(0)` is sent before arming. That shall work even without removing this code.